src/js/components/channels/twilio-channel-content.jsx

```diff
@@ -13,13 +14,10 @@ export class TwilioChannelContent extends Component {
             color: `#${settings.linkColor}`
         } : null;
 
-        if (isMobile.any) {
-            return <a href={ `sms:${phoneNumber}` }
-                      style={ styleOverride }>
-                       { phoneNumber }
-                   </a>;
-
-        }
-        return <span className='channel-content-value'>{ phoneNumber }</span>;
+        return <TelephoneInput ref='telInput'
+                               defaultCountry='ca'
```

Comment:
yeah.. this can't be `ca`... pretty sure we could do something with the user language client-side